### ISSUE TITLE
Pojmy jsou podtřídou ""^^xsd:integer

### ISSUE THREAD

**Issue by blcham** [opendata-mvcr/ontoGrapher#314]
V modelech slovníků je chyba u některých pojmů. Tyto pojmy jsou podtřídou ""^^xsd:integer. Příkladem je:
```
<https://slovník.gov.cz/legislativní/sbírka/329/2020/pojem/interní-náklady-isvs>
        rdfs:subClassOf  ""^^xsd:integer .
```
Týka se alepoň těchto pojmů:
<https://slovník.gov.cz/legislativní/sbírka/329/2020/pojem/interní-náklady-isvs>
<https://slovník.gov.cz/legislativní/sbírka/365/2000/pojem/referenční-rozhraní>
<https://slovník.gov.cz/legislativní/sbírka/329/2020/pojem/dodavatel>
<https://slovník.gov.cz/legislativní/sbírka/365/2000/pojem/správce-nebo-provozovatel-nebo-uživatel-informačního-systému-veřejné-správy>

**Comment by blcham** [comment]
regarding the commit, seems ok :)

**Comment by bindeali** [comment]
> Fixed the data - @bindeali can this be caused by OntoGrapher export?

Probably - I can see that the initial commit of (some of) these were done in mid June. I will do some testing and see if the problem is still there.

**Comment by bindeali** [comment]
UPDATE: seems highly unlikely that this sort of mistake could happen in OG now. The way subClassOf is utilized as of #313's [representation of connections in the model](https://github.com/opendata-mvcr/ontoGrapher/blob/f03d40526fb14106c4766190375d1cf0b0806381/src/queries/update/UpdateConnectionQueries.ts) is that 

1. Generalizations are [created](https://github.com/opendata-mvcr/ontoGrapher/blob/f03d40526fb14106c4766190375d1cf0b0806381/src/queries/update/UpdateConnectionQueries.ts#L227) via _subclassIRI_ rdfs:subClassOf _superclassIRI_ and there is no way to impose an xsd type in this situation.
2. Other connections are realized via a [standardized restriction](https://github.com/opendata-mvcr/ontoGrapher/blob/f03d40526fb14106c4766190375d1cf0b0806381/src/queries/update/UpdateConnectionQueries.ts#L26), i.e _iri_ rdfs:subClassOf [a owl: Restriction, ...]. The blank node part is always created in this situation.

In other words, there is very little chance (don't want to say "no chance" because I don't want to have to eat my words if I'm wrong :-) ) that the statement above could be generated by OG in the latest version.

**Comment by MichalMed** [comment]
@bindeali seems like OG is doing this again, see https://github.com/opendata-mvcr/ssp/commit/4b3a6080123d54d433b72ce006d8792e08682bdc, adding
 `d-sgov-ofn-úřední-desky-pojem:stránka-zveřejnění a <https://slovník.gov.cz/základní/pojem/typ-vlastnosti>;
  rdfs:subClassOf ""^^xsd:integer .`

Complete list of terms with this bug is here: https://slovník.gov.cz/sparql?default-graph-uri=&query=select+%3Fentity%2C+%3Fgraph+WHERE+%7bgraph+%3Fgraph+%7b%3Fentity+rdfs%3AsubClassOf+0%7d%7d&format=text%2Fx-html%2Btr&timeout=0&signal_void=on
